.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: "Build and push artifacts"
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Release artifacts
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set version
+        if: startsWith(github.ref, 'refs/tags')
+        run: |
+          tag_name="${GITHUB_REF##*/}"
+          tag_value="${tag_name:1}-ice36"
+          echo "omero.version=$tag_value" >> etc/local.properties
+      - name: Build and package
+        run: ./build.py build-dev release-all release-src
+      - name: Upload Release Assets
+        if: startsWith(github.ref, 'refs/tags')
+        run: |
+          set -x
+          assets=()
+          touch SHASUMS
+          touch MD5
+          for asset in ./target/*.zip; do
+            assets+=("$asset")
+            filename=$(basename -- "$asset")
+            sha=$(sha256sum $asset)
+            IFS=' ' read -r -a array <<< "$sha"
+            echo "${array[0]} $filename" >> SHASUMS
+            md5=$(md5sum $asset)
```

Comment:
Unless there is a specific driver, I would vote for removing md5 checksums which are superseded by sha256

Reply:
WFM. I kept them to match what is currently done as part of the release job